scale/docs/rest/job.rst

```diff
@@ -1050,3 +1050,120 @@ These services provide access to information about "all", "currently running" an
+|        "file_type": "SOURCE",                                                                                           |
+|        "file_size": 100,                                                                                                |
+|        "media_type": "text/plain",                                                                                      |
+|        "data_type": "",                                                                                                 |
+|        "meta_data": {...},                                                                                              |
+|        "url": null,                                                                                                     |
+|        "source_started": "2016-01-10T00:00:00Z",                                                                        |
+|        "source_ended": "2016-01-11T00:00:00Z",                                                                          |
+|        "data_started": "2016-01-10T00:00:00Z",                                                                          |
+|        "data_ended": "2016-01-11T00:00:00Z",                                                                            |
+|        "created": "2017-10-12T18:59:24.398334Z",                                                                        |
+|        "deleted": null,                                                                                                 |
+|        "last_modified": "2017-10-12T18:59:24.398379Z",                                                                  |
+|        "uuid": "",                                                                                                      |
+|        "is_deleted": false,                                                                                             |
+|        "workspace": {                                                                                                   |
+|            "id": 19,                                                                                                    |
+|            "name": "workspace-19"                                                                                       |
+|        },                                                                                                               |
+|        "countries": ["TCY", "TCT"],                                                                                     |
+|        "geometry" :null,                                                                                                |
+|        "center_point": null                                                                                             |
+|    }                                                                                                                    |
++-------------------------------------------------------------------------------------------------------------------------+
```

Comment:
This can return multiple files, so the docs should show an array with the paging fields. Applies to the other docs.